src/components/Sort.js

```diff
@@ -0,0 +1,46 @@
+import Component from './Component.js';
+
+class Sort extends Component {
```

Comment:
This class name is very confusing, I would consider it being more explicit. When you instantiate a ```new Sort``` in your ```app.js``` its confusing as to what that class is doing / sorting.